### ISSUE TITLE
C# code can't be ran in debugger on zorin os

### ISSUE THREAD

**Issue by Shadowblitz16** [OmniSharp/omnisharp-vscode#4194]
<!-- To prefill this information:
1. Open Visual Studio Code
2. Install C# plugin
3. Create .cs file with hello world example
4. Goto debug tab click create config -> .net core
5. generates a a config file with a bunch or warnings
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": ".NET Core Launch (console)",
            "type": "coreclr",
            "request": "launch",
            "WARNING01": "*********************************************************************************",
            "WARNING02": "The C# extension was unable to automatically decode projects in the current",
            "WARNING03": "workspace to create a runnable launch.json file. A template launch.json file has",
            "WARNING04": "been created as a placeholder.",
            "WARNING05": "",
            "WARNING06": "If OmniSharp is currently unable to load your project, you can attempt to resolve",
            "WARNING07": "this by restoring any missing project dependencies (example: run 'dotnet restore')",
            "WARNING08": "and by fixing any reported errors from building the projects in your workspace.",
            "WARNING09": "If this allows OmniSharp to now load your project then --",
            "WARNING10": "  * Delete this file",
            "WARNING11": "  * Open the Visual Studio Code command palette (View->Command Palette)",
            "WARNING12": "  * run the command: '.NET: Generate Assets for Build and Debug'.",
            "WARNING13": "",
            "WARNING14": "If your project requires a more complex launch configuration, you may wish to delete",
            "WARNING15": "this configuration and pick a different template using the 'Add Configuration...'",
            "WARNING16": "button at the bottom of this file.",
            "WARNING17": "*********************************************************************************",
            "preLaunchTask": "build",
            "program": "${workspaceFolder}/bin/Debug/<insert-target-framework-here>/<insert-project-name-here>.dll",
            "args": [],
            "cwd": "${workspaceFolder}",
            "console": "internalConsole",
            "stopAtEntry": false
        },
        {
            "name": ".NET Core Attach",
            "type": "coreclr",
            "request": "attach",
            "processId": "${command:pickProcess}"
        }
    ]
}
```

## Environment data
VS Code version: 1.51.0
C# Extension version: v1.23.5
OS: Zorin OS Core 15.3

## OmniSharp log
Not sure

## Steps to reproduce
See above?

## Expected  behavior
It should find it?

## Actual behavior
it fails with error code 127 and shows a popup dialog

**Comment by gregg-miskelly** [comment]
@Shadowblitz16 Do you just have a C# file, or do you have a project that builds? If you just have a file - this is expected. This extension doesn't really have much in the way of support for stand alone files.

If your have a project, please include your OminSharp log so that the folks who own the language service side of this extension can try and help you diagnose what is going on. It is one of the 'Output' tabs.

**Comment by Shadowblitz16** [comment]
I don't know where the omnisharp log is

**Comment by gregg-miskelly** [comment]
Yes, I tried to explain where to find it in my pervious comment, but let me try again -- In VS Code, there is an Output window. Switch to that. You should see a little drop down to select an output tab. Use that to pick 'OmniSharp log'.
